src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+$height_watch: 80vmin;
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: auto;
+}
+
+.watch {
```

Comment:
Create a separate file for watch styles
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/20fc0994-ffc0-484f-80cf-eb59f1f9beaf)

Reply:
done